engine/src/main/resources/assets/skins/behaviorEditor.skin

```diff
@@ -1,7 +1,7 @@
 {
     "inherit": "mainMenu",
     "text-shadowed": true,
-    "font": "default",
+    "font": "Noto",
```

Comment:
I was wondering if it should be some sans version as it is more similar to the "default" font, I'll update it.